cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/util/YamlUtils.java

```diff
@@ -0,0 +1,45 @@
+package cloud.foundry.cli.crosscutting.util;
+
+import cloud.foundry.cli.crosscutting.beans.ApplicationBean;
+import cloud.foundry.cli.crosscutting.beans.ApplicationManifestBean;
+import cloud.foundry.cli.crosscutting.beans.ServiceInstanceSummaryBean;
+import cloud.foundry.cli.crosscutting.beans.SpaceDevelopersBean;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.nodes.Tag;
+import org.yaml.snakeyaml.representer.Representer;
+
+import java.util.HashMap;
+
+public class YamlUtils {
+
+    /**
+     * Factory function creating a Yaml object with common options. Ensures a consistent output format.
+     * @return Yaml object preconfigured with proper options
+     */
+    public static Yaml createDefaultYamlParser() {
```

Comment:
This looks like creating a YAML dumper, not parser. Or can it serve for both purposes? Anyway, please consider renaming the class.